components/shared/ViewSidebar.vue

```diff
@@ -32,14 +33,6 @@ const props = defineProps<{
 
 const scrolled = ref(false);
 
-/** To hide the scroll indicator when the user scrolls */
-const handleScroll = (event: Event) => {
```

Comment:
With this removed, I don't see anything updating `scrolled.value`. I can confirm this on UI too - once you scroll on the sidebar, the icon doesn't disappear anymore. Previously, this was handled by `@scroll="handleScroll"` in the sidebar div.